### PR TITLE
test: added tests for languageSet Model

### PR DESCRIPTION
**Pull request by akashgp09**
### Problem

Tests for languageSet Model was missing

### Solution

Added Tests for languageSet Model

Screenshot from 2021-04-26 17-33-25

### CODE REVIEW

**Comment by akashgp09** [comment]
> Could I ask you to change from promise.then to async/await syntax?

Sure, I will make the changes soon :)

File: test/testLanguageSet.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	};
+
+	afterEach(
+		() => truncateTables(bookshelf, [
+			'bookbrainz.language_set',
+			'musicbrainz.language'
+		])
+	);
+
+	it('should return a JSON object with correct keys when saved', async () => {
```
Comment (akashgp09) [inline test/testLanguageSet.js:56]: @MonkeyDo travis CI build check is failing on this PR due to this test. But when i run the test locally, it is passing successfully 🤔

Comment (MonkeyDo) [reply test/testLanguageSet.js:56]: I'm guessing you're missing a call to `truncateTables` before running the tests, basically exactly the same as `afterEach` but using `beforeAll`.

The reason is probably that some other test somewhere already creates a LanguageSet with id=1 and does not do an appropriate cleanup afterwards.

Either that or some race condition; it looks like everywhere in the tests we should be using `await truncateTables(…`, and perhaps some other test's cleanup step is not yet finished when this test runs…

**Comment by akashgp09** [comment]
@MonkeyDo seems like i found the imposter. In the language tests, we saved some data in `language_set` table but never deleted the data again. Hence truncating that table solves the issue.

**Review by MonkeyDo — APPROVED** [review]
Thanks for adding thes !

**Pull request merged**